requirements_all.txt

```diff
@@ -165,6 +165,9 @@ apns2==0.3.0
 # homeassistant.components.aqualogic
 aqualogic==1.0
 
+# homeassistant.components.air_quality.ampio
```

Comment:
Please move, `git mv`, the air quality platform to an `ampio` component package. That's our new module structure.

```
homeassistant/components/ampio/__init__.py
homeassistant/components/ampio/air_quality.py
```

Just add a docstring in the init module. `.coveragerc` and requirements need to be updated after the move.

Reply:
Done